# This is a BitKeeper generated diff -Nru style patch.
#
# ChangeSet
#   2004/12/27 21:08:40-08:00 davem@nuts.davemloft.net 
#   [SPARC64]: Fix typo in previous change, s/IS_SOCK/S_ISSOCK/
#   
#   Signed-off-by: David S. Miller <davem@davemloft.net>
# 
# arch/sparc64/solaris/timod.c
#   2004/12/27 21:08:08-08:00 davem@nuts.davemloft.net +1 -1
#   [SPARC64]: Fix typo in previous change, s/IS_SOCK/S_ISSOCK/
# 
diff -Nru a/arch/sparc64/solaris/timod.c b/arch/sparc64/solaris/timod.c
--- a/arch/sparc64/solaris/timod.c	2005-01-04 17:51:52 -08:00
+++ b/arch/sparc64/solaris/timod.c	2005-01-04 17:51:52 -08:00
@@ -921,7 +921,7 @@
 	ino = filp->f_dentry->d_inode;
 	if (!ino) goto out;
 
-	if (!IS_SOCK(ino->i_mode) &&
+	if (!S_ISSOCK(ino->i_mode) &&
 		(imajor(ino) != 30 || iminor(ino) != 1))
 		goto out;
 
